packages/manager/apps/telecom/src/app/telecom/pack/slots/voipLine/activation/pack-voipLine-activation.controller.js

```diff
@@ -301,10 +301,12 @@ export default class PackVoipLineActivationCtrl {
     this.orderPending = true;
     const data = [];
     this.selectedPhones.forEach((line) => {
-      if (line.needShipping) {
-        data.push(
-          angular.extend({ hardwareName: line.name }, this.getTransporter()),
-        );
+      if (line.needShipping && line.quantity > 0) {
```

Comment:
As @frenautvh said in previous comment, it could be : 

```suggestion
      if (line.needShipping) {
```

If `line.quantity === 0` the `for` loop won't trigger.

Reply:
OK